src/app/routing/pages/generic-page/demo/demo.component.html

```diff
@@ -1,2 +1,6 @@
 <h2 class="govuk-heading-m">DEMO PAD</h2>
-<p>This is a place to test out components</p>
\ No newline at end of file
+<p>This is a place to test out components</p>
+
+
+<app-assembly-viewer [templates]="templates"
```

Comment:
Might be worth modifying the following file:
../rpa-em-show/src/app/routing/routing.module.ts

See example modification that I've done with the Doc Viewer:
const routes: Routes = [
    {
        path: '',
        component: GenericPageComponent,
        children: [
            {
                path: '',
                component: DemoComponent,
                canActivate: [AuthGuardService],
            }
        ]
    },
    {
        path: 'document-viewer',
        component: GenericPageComponent,
        children: [
            {
                path: '',
                component: DocViewerComponent,
                canActivate: [AuthGuardService],
            }
        ]
    }

You'll also need to add an item to the navigation:
../rpa-em-show/src/app/header/header.component.ts
e.g - as I have done for Document Viewer
this.navItems = [
            {
                text: 'Home',
                href: '/',
                active: true
            },
            {
                text: 'Document Viewer',
                href: '/document-viewer',
                active: false
            }
        ];